### PR TITLE
Forrr connect UI

### PR DESCRIPTION
**Pull request by mobimation-gunnar**
(Thanks for sending a pull request! Please make sure you click the link above to view the [contribution guidelines](CONTRIBUTING.md), then fill out the blanks below.)

---

UI implementation according to PDF for Connect Once service.

...

It opens up the possibility to begin closing some issues.
...

**Any relevant logs, error output, etc?**

...

Part of changes that affect af-connect-module as well
...

**Where has this been tested?**
 - Macbook Pro 15 (2017)
 - **OS:** Catalina 10.15.7
 - **Browser:** Chrome
 - **Version:** 91.0.4472.77 (Official Build) (x86_64)

### CODE REVIEW

File: app/public/js/main.bundle.js [hunk 1]
```diff
@@ -11,6 +10,7 @@ let config = {
 };
 
 let getSessionToken = () => {
+  console.log("GF:getSessionToken");
```
Comment (ixuz) [inline app/public/js/main.bundle.js:13]: Remove this console log

File: app/public/js/main.bundle.js [hunk 2]
```diff
@@ -154,29 +160,35 @@ window.refreshShareButton = function refreshShareButton() {
 };
 
 window.secrecyAgreement = function secrecyAgreement() {
+  console.log("secrecyAgreement");
```
Comment (ixuz) [inline app/public/js/main.bundle.js:163]: Remove this console log

File: app/public/js/main.bundle.js [hunk 3]
```diff
@@ -154,29 +160,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   window.refreshShareButton();
 };
 
 window.transferAgreement = function transferAgreement() {
+  console.log("transferAgreement");
```
Comment (ixuz) [inline app/public/js/main.bundle.js:168]: Remove this console log

File: app/public/js/main.bundle.js [hunk 4]
```diff
@@ -154,29 +160,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   window.refreshShareButton();
 };
 
 window.reviewAgreement = function reviewAgreement() {
+  console.log("reviewAgreement");
```
Comment (ixuz) [inline app/public/js/main.bundle.js:173]: Remove this console log

File: app/public/js/main.bundle.js [hunk 5]
```diff
@@ -154,29 +160,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   window.refreshShareButton();
 };
 
 window.openTermsAgreement = function openTermsAgreement() {
+  console.log("openTermsAgreement");
```
Comment (ixuz) [inline app/public/js/main.bundle.js:178]: Remove this console log

File: app/public/js/main.bundle.js [hunk 6]
```diff
@@ -154,29 +160,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   $("#termsModal").show();
   $("#termsAgreement").prop("checked", !$("#termsAgreement").prop("checked"));
 };
 
 window.onTermsAgreement = function onTermsAgreement() {
+  console.log("onTermsAgreement");
```
Comment (ixuz) [inline app/public/js/main.bundle.js:184]: Remove this console log

File: app/public/js/main.bundle.js [hunk 7]
```diff
@@ -154,29 +160,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   $("#termsModal").hide();
   $("#termsAgreement").prop("checked", true);
   window.refreshShareButton();
 };
 
 window.onTermsCancel = function onTermsCancel() {
+  console.log("onTermsCancel");
```
Comment (ixuz) [inline app/public/js/main.bundle.js:191]: Remove this console log

File: app/public/js/main.js [hunk 8]
```diff
@@ -10,6 +9,7 @@ let config = {
 };
 
 let getSessionToken = () => {
+  console.log("GF:getSessionToken");
```
Comment (ixuz) [inline app/public/js/main.js:12]: Remove this console log

File: app/public/js/main.js [hunk 9]
```diff
@@ -153,29 +159,35 @@ window.refreshShareButton = function refreshShareButton() {
 };
 
 window.secrecyAgreement = function secrecyAgreement() {
+  console.log("secrecyAgreement");
```
Comment (ixuz) [inline app/public/js/main.js:162]: Remove this console log

File: app/public/js/main.js [hunk 10]
```diff
@@ -153,29 +159,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   window.refreshShareButton();
 };
 
 window.transferAgreement = function transferAgreement() {
+  console.log("transferAgreement");
```
Comment (ixuz) [inline app/public/js/main.js:167]: Remove this console log

File: app/public/js/main.js [hunk 11]
```diff
@@ -153,29 +159,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   window.refreshShareButton();
 };
 
 window.reviewAgreement = function reviewAgreement() {
+  console.log("reviewAgreement");
```
Comment (ixuz) [inline app/public/js/main.js:172]: Remove this console log

File: app/public/js/main.js [hunk 12]
```diff
@@ -153,29 +159,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   window.refreshShareButton();
 };
 
 window.openTermsAgreement = function openTermsAgreement() {
+  console.log("openTermsAgreement");
```
Comment (ixuz) [inline app/public/js/main.js:177]: Remove this console log

File: app/public/js/main.js [hunk 13]
```diff
@@ -153,29 +159,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   $("#termsModal").show();
   $("#termsAgreement").prop("checked", !$("#termsAgreement").prop("checked"));
 };
 
 window.onTermsAgreement = function onTermsAgreement() {
+  console.log("onTermsAgreement");
```
Comment (ixuz) [inline app/public/js/main.js:183]: Remove this console log

File: app/public/js/main.js [hunk 14]
```diff
@@ -153,29 +159,35 @@ window.refreshShareButton = function refreshShareButton() {
...
   $("#termsModal").hide();
   $("#termsAgreement").prop("checked", true);
   window.refreshShareButton();
 };
 
 window.onTermsCancel = function onTermsCancel() {
+  console.log("onTermsCancel");
```
Comment (ixuz) [inline app/public/js/main.js:190]: Remove this console log

File: app/views/partials/consentForm.ejs [hunk 15]
```diff
@@ -1,202 +1,380 @@
...
+    <div style="margin: 30px 0px 0px 70px;">
+        <button class="back-button" onclick="showPage(2);clearBoxes();">Tillbaka</button>
+        <button id="shareButton" class="fwd-button" style="padding: 11px 100px;" onclick="onConsent();" disabled>
+            Godkänn
+        </button>
+    </div>
 </div>
+</body>
+<script>
+    function showPage(number) {
```
Comment (ixuz) [inline app/views/partials/consentForm.ejs:335]: Move this function to `app/public/js/main.js`

File: app/views/partials/consentForm.ejs [hunk 16]
```diff
@@ -1,202 +1,380 @@
...
+                      profile.style.display = "block";
+                    else
+                      profile.style.display = "none";
+                    x++;
+                }
+            });
+        }
+    }
+    // Clear consent when moving from consent page
+    function clearBoxes() {
```
Comment (ixuz) [inline app/views/partials/consentForm.ejs:358]: Move this function to `app/public/js/main.js`

File: app/views/partials/consentForm.ejs [hunk 17]
```diff
@@ -1,202 +1,380 @@
...
+        window.document.getElementById("secrecyAgreement").checked=false;
+        window.document.getElementById("transferAgreement").checked=false;
+        window.document.getElementById("reviewAgreement").checked=false;
+        window.document.getElementById("termsAgreement").checked=false;
+        refreshShareButton();
+    }
+
+    function consent() {
```
Comment (ixuz) [inline app/views/partials/consentForm.ejs:366]: Move this function to `app/public/js/main.js`

**Review by ixuz — CHANGES_REQUESTED** [review]
- Remove `app/public/img/graphics`
- Remove `app/views/pages/index-ny.ejs`
-

File: app/public/js/main.bundle.js (hunk 2 above)
Comment (mobimation-gunnar) [reply app/public/js/main.bundle.js:163]: Done.

File: app/public/js/main.bundle.js (hunk 4 above)
Comment (mobimation-gunnar) [reply app/public/js/main.bundle.js:173]: Done.

File: app/public/js/main.js (hunk 13 above)
Comment (mobimation-gunnar) [reply app/public/js/main.js:183]: Done.

File: app/public/js/main.js (hunk 14 above)
Comment (mobimation-gunnar) [reply app/public/js/main.js:190]: Done.

File: app/views/partials/consentForm.ejs (hunk 16 above)
Comment (mobimation-gunnar) [reply app/views/partials/consentForm.ejs:358]: Done.

File: app/views/partials/consentForm.ejs (hunk 17 above)
Comment (mobimation-gunnar) [reply app/views/partials/consentForm.ejs:366]: Done.  <script> section removed from consentForm.ejs

**Comment by mobimation-gunnar** [comment]
Pushed update

**Review by ixuz — APPROVED** [review]

**Pull request merged**